lib/zold/remotes.rb

```diff
@@ -62,6 +63,10 @@ def all
       def iterate(_)
         # Nothing to do here
       end
+
+      def mtime
```

Comment:
@emilianodellacasa since you add this, why do you need `attr_reader`? Unless it's defined in a different class.